lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -435,20 +394,19 @@ void IMConstPropPass::markBlockExecutable(Block *block) {
   }
 }
 
-void IMConstPropPass::markWireOrRegOp(Operation *wireOrReg) {
+void IMConstPropPass::markWireOp(Operation *wire) {
```

Comment:
This should probably take a WireOp as a paremeter instead of using `getResult(0)`